codes/2529_cpp_yji0828.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+using namespace std;
+
+int k;
+char sign[11];
+int num[11];
+bool check[11];
+vector<string> answers;
+string maximum, minimum;
+
+bool check_one(char a, char b, char oper);
+void comb(int index, string num);
+
+int main() {
+    cin >> k;
+
+    for (int i = 0; i < k; i++)
+        cin >> sign[i];
+
+    comb(0, "");
+    sort(answers.begin(), answers.end());
+    cout << answers.back() << '\n' << answers.front();
+
+    return 0;
+}
+
+bool check_one(char a, char b, char oper) {
+    if (oper == '<') {
+        if (a > b) return false;
```

Comment:
if를 한줄로 적지말아주세요. 이곳 말고 다른 부분도 다 고쳐주세요.
```suggestion
        if (a > b)
            return false;
```

Reply:
반영하겠습니다!